english/find-exit/into-mountains/boulder/boulders.md

```diff
@@ -0,0 +1,7 @@
+As you push against some of the boulders, they fall downwards into the
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.